### PR TITLE
Help in feature 10870 newadmin playlists

### PR DESCRIPTION
**Pull request by rubenrua**

### CODE REVIEW

File: src/Pumukit/NewAdminBundle/Resources/views/PlaylistMultimediaObject/addModal.html.twig [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<div class="modal-header">
+  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+    <span aria-hidden="true">&times;</span>
+  </button>
+  <h4 class="modal-title" id="myModalLabel">{% trans %}Add to playlist{% endtrans %}</h4>
+</div>
+<div class="modal-body">
+  
+  {% if multimediaObjet %}
```
Comment (mreyk) [inline src/Pumukit/NewAdminBundle/Resources/views/PlaylistMultimediaObject/addModal.html.twig]: Typo

File: src/Pumukit/NewAdminBundle/Resources/views/PlaylistMultimediaObject/addModal.html.twig [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    {% trans with {'%title%': multimediaObjet.title|default(ids) } %}Add the object "%title%" to the playlist{% endtrans %}:
```
Comment (mreyk) [inline src/Pumukit/NewAdminBundle/Resources/views/PlaylistMultimediaObject/addModal.html.twig]: Typo

File: src/Pumukit/NewAdminBundle/Controller/PlaylistMultimediaObjectController.php [hunk 3]
```diff
@@ -293,4 +293,100 @@ protected function enableFilter()
...
+        $repoMms = $this->getDoctrine()
+            ->getRepository('PumukitSchemaBundle:MultimediaObject');
+
+        $series = $repoSeries->createQueryBuilder()
+          ->field('type')->equals(Series::TYPE_PLAYLIST)
+          ->sort('public_date', -1)
+          ->getQuery()
+          ->execute();
+
+        $multimediaObjet = $request->get('id') ? $repoMms->find($request->get('id')) : null;
```
Comment (mreyk) [inline src/Pumukit/NewAdminBundle/Controller/PlaylistMultimediaObjectController.php]: Typo

File: src/Pumukit/NewAdminBundle/Controller/PlaylistMultimediaObjectController.php [hunk 4]
```diff
@@ -293,4 +293,100 @@ protected function enableFilter()
...
+        
+        $count = 0;
+        if ($request->get('ids')) {
+            $ids = json_decode($request->get('ids'));
+            $count = count($ids);
+        }
+
+        return array(
+            'series' => $series,
+            'multimediaObjet' => $multimediaObjet,
```
Comment (mreyk) [inline src/Pumukit/NewAdminBundle/Controller/PlaylistMultimediaObjectController.php]: Typo

**Comment by Alfro** [comment]
Thanks.

**Pull request merged**